.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.4
```

Comment:
why this version?

Reply:
No specific reason. 

[Stable releases](https://www.ruby-lang.org/en/downloads/)
* Ruby 2.5.1
* Ruby 2.4.4
* Ruby 2.3.7

Which one to use, no clue (yet)

Reply:
@kinduff I have looked into this. 
`gem 'axlsx', '~> 2.0.1'` supports [Ruby Version: 2.2.7, 2.3.4, 2.4.1](https://github.com/randym/axlsx)
When running ruby version `2.4.1` with `axlsx 2.0.1` I received some deprecated warning:

```
/Users/_/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-2.0.1/lib/axlsx/util/validators.rb:88: warning: constant ::Fixnum is deprecated

/Users/_/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-2.0.1/lib/axlsx/util/validators.rb:110: warning: constant ::Fixnum is deprecated

/Users/_/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-2.0.1/lib/axlsx/util/simple_typed_list.rb:146: warning: constant ::Fixnum is deprecated
```

On RubyGems axlsx is only available as [3.0.0.pre](https://rubygems.org/gems/axlsx/versions/3.0.0.pre)  and the Readme of the project states "latest version 3.0.0", so from that I updated the ruby version to 2.4.1 and use 3.0.0.pre for axlsx.

The thing with the ruby-version is probably, that the latest version of ruby is not yet adopted by the community.

Using the latest version, would be good if the project would not have the dependency. I guess that a another discussion :)